src/components/Books.jsx

```diff
@@ -0,0 +1,21 @@
+import Form from './Form';
+import Button from './shared/Button';
+
+const Books = () => (
+  <div>
+    <h2>List of Books</h2>
+    <ul>
+      <li>
+        Fight Hunger by Nim
+        <Button value="Remove" />
+      </li>
+      <li>
+        Fight Hunger by Nim
+        <Button value="Remove" />
```

Comment:
- [ ] This remove button should be in the Book component as that is where the remove event will be triggered and handled. Kindly move it to the component.